### ISSUE TITLE
zero_objective! not necessary

### ISSUE THREAD

**Issue by tomerarnon** [sisl/NeuralVerification.jl#51]
I'm pretty sure solving a problem without specifying an objective simply solves a feasibility problem, so it should be equivalent to specifying @objective(m, Max, 0), which is what we do now.

**Comment by changliuliu** [comment]
Yes. The purpose of having zero_objective! is just to make it more clear that we are solving a feasibility problem.

**Comment by tomerarnon** [comment]
Ah, I see. In that case we can be even more explicit with something like `feasibility_problem!(model)`

**Comment by mykelk** [comment]
Even better!
